### PR TITLE
January 2017 Revisions

### PR DESCRIPTION
**Pull request by DrewRomanyk**
*** THIS HAS NOT BEEN APPROVAL BY BOARD AND GENERAL, DO NOT MERGE ***

This Revision is designed to make the bylaws streamlined with the Board Order system and less confusing in matters dealing with elections and which positions are supposed to be elected upon and which aren't.

The Purchase Proposal System has been moved to the Board Order system

I Also would like to add a wiki to provide a clear and short format for people to understand certain processes that aren't well known in LAN.

- [x] Pledge process
- [x] List of members by class
- [x] List of officers and positions by year
- [x] Voting process FAQ

- [x] Need to add clause for Nominated to be semesterly

### CODE REVIEW

**Comment by arrdem** [comment]
🙏

File: bylaws.md [hunk 1]
```diff
@@ -85,36 +87,23 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
...
-            3. If the proposal is not vetoed, the process will continue as described below.
-          4. The Treasurer will briefly present the purchase proposal at a general meeting.
-            1. The general membership may initiate the veto process by support of three active members.
-            2. The general membership may veto the purchase proposal by a simple majority of those present. If vetoed, the proposal must be recreated and must go through the entire purchase proposal process again.
-          5. The purchase proposal is executed as described in the proposal form.
-          6. The Treasurer will follow up with the proposal sponsor to ensure it is properly and fully executed.
-          7. If the purchase is not properly executed as described in the purchase proposal, the Treasurer will take steps to correct the situation (including, but not limited to, cancellation of the purchase proposal).
 5. Terms of Office:
-    1. Officers shall be elected towards the end of the spring semester on a yearly basis.
+    1. Elected Officer posistions shall be elected on towards the end of the spring semester on a yearly basis.
```
Comment (aaaanderson) [inline bylaws.md]: This sentence looks ungrammatical to me, and not explicit about the time requirement.

File: bylaws.md [hunk 2]
```diff
@@ -126,25 +115,27 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
 	   If no majority is achieved or if no vote achieves a majority then all standing candidates are discarded and the election for that position must be re-run.
 	   Candidates may be re-nominated.
     2. Board motions shall require a 2/3rds vote of the Board members to pass.
-7. Board meetings:
+7. Nominating:
+    1. For all nominated positions, they shall be chosen by the Board officer to whom the nominated position reports, and must be approved by a vote of 2/3rds of the Board.
```
Comment (aaaanderson) [inline bylaws.md]: This sentence could be reworded to be more grammatical + easier to understand.

One candidate for replacement could be:

1. All nominated positions shall be chosen by the Board officer to whom the nominated position reports.
2. All nominated positions must be approved by a 2/3rds vote of the Board.

File: bylaws.md [hunk 3]
```diff
@@ -126,25 +115,27 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
...
 	   Upon a vote of removal or suspension of an officer, the normal vacancy policy immediately comes into effect.
-6. Powers of the Board
-   1. In its executive capacity, the Board is authorized to plan and conduct Rush as well as other events; to set and collect dues; take disciplinary action against Members under established Policies and Codes of Conduct; to accept new Pledges into the Fraternity and conduct all business necessary and proper to the function and continuance of the Fraternity that are otherwise not specified in these bylaws.
-   2. The Board may make and impose no Rule, Policy or Code of Conduct upon the Membership of the Fraternity save by a 2/3rds vote of the Membership taken at a General or Special meeting of the Fraternity.
+10. Powers of the Board
+	1. In its executive capacity, the Board is authorized to plan and conduct Rush as well as other events; to set and collect dues; take disciplinary action against Members under established Policies and Codes of Conduct; to accept new Pledges into the Fraternity and conduct all business necessary and proper to the function and continuance of the Fraternity that are otherwise not specified in these bylaws.
```
Comment (aaaanderson) [inline bylaws.md]: Structurally, I appreciate that this clause explicitly delineates Board powers.

However, the subclause discussing its disciplinary responsibilities is confusing. Is Standards Board given its disciplinary responsibility as a matter of delegation from the Executive Board, or is Standards Board an "equal" body but with radically different responsibilities?

I prefer the latter interpretation. I'm not sure how exactly to make this clearer, though; perhaps moving the section about discipline to a new clause, that discusses the relationship of Exec and Standards, is for the best.

File: bylaws.md [hunk 4]
```diff
@@ -168,8 +159,13 @@ All members must provide and monitor an email address which will serve as the so
 Emails from the board must be sent from an @texaslan.org (http://texaslan.org/) email account.
 Any communications through social media must follow an official announcement in email form.
 
-**Article X. Amendments**
+**Article X. Board Orders**
+
+Processes that aren't required to be in the Bylaws shall be placed in the [Board Orders Document](./board_orders.md), to where it defines extra processes these bylaws must follow.
```
Comment (aaaanderson) [inline bylaws.md]: I would redraft this clause entirely; it needs more detail. Here's a suggestion, subject to how the process actually works.

* Processes and directives which are not Bylaws but which are required by the sitting Board are referred to as Board Orders.
* Board Orders may be enacted by a 2/3rds vote of the Board.
* Board Orders must be re-enacted upon election of a new Board.
* Board Orders may be repealed by a 2/3rds vote of the Board.
* Board Orders shall be promptly posted in the [Board Orders Document](./board_orders.md) and communicated to the General Membership through the official communications channels.

File: bylaws.md [hunk 5]
```diff
@@ -168,8 +159,13 @@ All members must provide and monitor an email address which will serve as the so
...
+These Board Orders can be altered by a vote of 2/3rds of the Board.
+
+**Article XI. Amendments**
 
 Changes to these bylaws may proposed and drafted by the Board, which is will then present the proposed changes to the general membership in a full meeting called for that purpose.
-Additionally, once every academic year in the spring semester, the current Board shall meet with the general membership to discuss the current bylaws and determine if any changes must be made.
+Additionally, once every academic year in the spring semester, the current Board shall meet with the general membership to discuss the current bylaws and board orders, and determine if any changes must be made.
```
Comment (aaaanderson) [inline bylaws.md]: Capitalize all instances of an introduced and defined term. In this case: "Bylaws and Board Orders"

**Comment by aaaanderson** [comment]
Great changes! I think there are just one or two hanging loose ends, I'll comment on the ones I noticed.

File: bylaws.md [hunk 6]
```diff
@@ -103,10 +103,11 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
 	   The Treasurer’s primary responsibility shall be to present a written statement of accounts and transactions to the board on a semesterly basis.
 	   In the case of financial difficulty, the Treasurer shall be responsible for notifying the Board immediately and assisting in the design and execution of recovery efforts.
 5. Terms of Office:
-    1. Elected Officer posistions shall be elected on towards the end of the spring semester on a yearly basis.
+    1. Elected Officer positions shall be elected upon on the 2nd to last scheduled meeting of the academic year.
```
Comment (aaaanderson) [inline bylaws.md]: This sentence still isn't grammatical.

File: bylaws.md [hunk 7]
```diff
@@ -103,10 +103,11 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
...
 	   In addition, an officer who is unable to complete the second semester of their duties shall be replaced in an election at the end of the fall semester.
-    2. A Member may serve in an  elected Officer position for only one full term.
-	   Members may be nominated for the same position repeatedly, unless there is compelling organizational reason to do so such re-appointments shall be frowned upon.
+    2. A Member may serve in an elected Officer position for only one full term.
+	   Members may be nominated for the same position repeatedly unless there is a compelling organizational reason to do so.
```
Comment (aaaanderson) [inline bylaws.md]: The meaning of this line is unclear. I read it as:

first half: "Members may be nominated for the same position repeatedly"
second half: "members may be nominated for the same position if there is a compelling organizational reason to do so"
linked by: "unless" implying a contrast between the halves"

Is the intent: "Members may be nominated and elected to a position they have previously been elected to if there is a compelling organizational reason to do so"?

You may wish to nail down how exactly we want to treat repeat elected candidates who served a partial term (for instance: Steven Diaz's tenure as VP Service after the special election, then again the next academic year, if I remember things correctly)

File: bylaws.md (hunk 6 above)
Comment (DrewRomanyk) [reply bylaws.md]: Would this be better?

During the 2nd to last scheduled meeting of the academic year, all elected Officer positions shall be voted in.

Comment (aaaanderson) [reply bylaws.md]: Yes, that would be grammatical and understandable.

Another grammatical example would be replacing "upon on" with "during".

**Pull request merged**